THEOplayer-Connector-Conviva.podspec

```diff
@@ -16,6 +16,7 @@ Pod::Spec.new do |s|
   s.platforms    = { :ios => "13.0", :tvos => "13.0" }
 
   s.source_files = 'Code/Conviva/Source/**/*'
+  s.resource_bundles = {'THEOplayer-Connector-Conviva' => ['version.json']}
```

Comment:
I think we need to do the same inclusion for the SPM packages too.
(and I think when you access the resources through SPM vs Cocoapods, something need to be changed when looking for the path. I think @Dev1an had some issues with this in the past)